### ISSUE TITLE
Reminders node

### ISSUE THREAD

**Issue by danielepolencic** [uasabi/noderosso#23]
TL;DR: Create a node for node-red that creates a reminder in Google calendar.

Create a script that:

- Retrieves all events from Google Calendar from yesterday.
- If any of those events start with "REMINDER" in the title, move the event to today at the same time.

The node is trigger by the following message:

```json
{
  "topic": "TICK.V1"
}
```

The node should provide a way to authenticate against a google calendar.

Most of the work necessary to write this node is available in this node-red node: https://github.com/mdhom/node-red-contrib-google-oauth-calendar
So why not use that one?

- The authentication code works only for node-red installation that starts with `/`. Since this starts with `/admin`, it does not work. You can copy and use that page, though.
- I cannot import the JSON credentials. There's a bug somewhere because if I paste the JSON the node suggests that it is malformed (it is not).
- We use only a fraction of the functionality.

## Google, Auth and credentials.

I will share the credentials for the app with you. Just ping me. Your email address has to be added to the allowlist as well.
